provider/doc_edits.go

```diff
@@ -68,3 +71,18 @@ var removeBetaFromDescriptionField = tfbridge.DocsEdit{
 		return content, nil
 	},
 }
+
+// Example HCL seems to employ `%{random_suffix}` which trips up the example conversion. Substitute
+// it out to an actual random suffix.
+var substituteRandomSuffix = (func() tfbridge.DocsEdit {
+	pattern := regexp.MustCompile(regexp.QuoteMeta("%{random_suffix}"))
+	return tfbridge.DocsEdit{
+		Path: "*",
+		Edit: func(path string, content []byte) ([]byte, error) {
+			return pattern.ReplaceAllFunc(content, func([]byte) []byte {
+				//nolint:gosec
+				return []byte(fmt.Sprintf("_%d", rand.Intn(100000)))
```

Comment:
I guess I need determinism here since it goes into the builds. Let me look at fixing the seed.